README.md

```diff
@@ -17,3 +17,5 @@
 <br>
 
 TypeScript client for dYdX (v3 API).
+<br />
+[Typescript client documentation link](https://docs.dydx.exchange/#typescript-client)
```

Comment:
I think you can just do `(#typescript-client)`
https://gist.github.com/asabaylus/3071099